### DISCUSSION TITLE
issue with libraries path

### DISCUSSION THREAD

**Discussion by rpoltera** [manyfold3d/manyfold#2785] [Support] [answered]
I must be missing the cause of my issue I have created a /mnt/media/3d in my unraid server and pointed to /libraries 
![Screenshot 2024-09-24 214805](https://github.com/user-attachments/assets/51d2e753-a203-4de7-b52a-7accc347be3c)

![Screenshot 2024-09-24 214424](https://github.com/user-attachments/assets/5177adeb-e6ad-4ad1-b1e9-2f35c9d187d5)
as I have tried multiple other options and paths but figured would try your example but when I point to /libraries/models in the UI I gett an error "Path cannot be a privileged system path" 
![Screenshot 2024-09-24 214650](https://github.com/user-attachments/assets/1fe45525-a103-4794-b91a-fcdaaca107a2)

**Comment by Floppy** [comment]
Hm, that's not right, that should only come up on (as it says) system paths. 

Does /mnt/media/3d/models exist? Wondering if it's a different error being masked...

I'll take a look into it.

**Reply by Floppy** [reply]
yep, OK, I'm seeing the same thing. That's a bug. Will fix!

**Reply by Floppy** [reply]
CRAP, I bet it's because I treat /lib as a system path, and /libraries starts with /lib. OMG what a fool I am.

**Reply by Floppy** [reply, marked as answer]
OK, v0.80.1 is out and should fix this. Sorry!

https://github.com/manyfold3d/manyfold/releases/tag/v0.80.1

**Comment by rpoltera** [comment]
Not sure what you did it is working now thanks it can be closed
